src/components/Select.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <fieldset class="f-select" @click="focusInput">
+    <select
+      :required="required"
+      :value="value"
+      ref="input"
+      @change="$emit('update:value', $event.target.value)" >
```

Comment:
why do we need this line? does it not update its values automatically?

Reply:
This line is necessary since the input component wraps more than one element. documentation: https://vuejs.org/v2/guide/components-custom-events.html#sync-Modifier